src/site/content/en/blog/nfc/index.md

```diff
@@ -0,0 +1,911 @@
+
+Writing a text record to an NFC tag is straightforward. Simply pass a string to
+the `write()` method.
+
+```js
+const writer = new NDEFWriter();
+writer.write(
+  "Hello World"
+).then(() => {
+  console.log("Message written.");
+}).catch(error => {
+  console.log(`Write failed :-( try again: ${error}.`);
+});
+```
+
+To write a URL record to an NFC tag, pass a dictionary that represents an NDEF
+message to the `write()` method. In the example below, the NDEF message is a
+dictionary with a `records` key. Its value is an array of records - in this
+case, a URL record defined as an object with a `recordType` key set to `"url"`
+and a `data` key set to the URL string.
+
+```js
+const writer = new NDEFWriter();
+writer.write({
+  records: [{ recordType: "url", data: "https://w3c.github.io/web-nfc/" }]
+}).then(() => {
+  console.log("Message written.");
+}).catch(error => {
+  console.log(`Write failed :-( try again: ${error}.`);
+});
+```
+
+It is also possible to write multiple records to an NFC tag.
+
+```js
+const writer = new NDEFWriter();
+writer.write({ records: [
+    { recordType: "url", data: "https://w3c.github.io/web-nfc/" },
+    { recordType: "url", data: "https://web.dev/web-nfc/" }
+]}).then(() => {
+  console.log("Message written.");
+}).catch(error => {
+  console.log(`Write failed :-( try again: ${error}.`);
+});
+```
+
+{% Aside %}
+The [cookbook](#cookbook) contains many examples of how to write other types of
+NDEF records.
+{% endAside %}
+
+If an NFC tag should be read while performing a NFC write operation, set the
+`ignoreRead` property to `false` in the options passed to the `write()` method.
+
+```js
+const reader = new NDEFReader();
+reader.scan().then(() => {
+
+  reader.onreading = event => {
+    // TODO: Handle incoming NDEF messages.
+  };
+
+  const writer = new NDEFWriter();
+  return writer.write("Writing data is fun!", { ignoreRead: false });
+
+}).catch(error => {
+  console.log(`Write failed :-( try again: ${error}.`);
+});
+```
+
+If the NFC tag contains an NDEF message that is not meant to be overwritten, it
+is possible to set the `overwrite` property to `false` in the options passed to
+the `write()` method. In that case, the returned promise will reject if an NDEF
+message is already stored in the NFC tag.
+
+```js
+const writer = new NDEFWriter();
+writer.write("Writing data on an empty NFC tag is fun!", { overwrite: false })
+.then(() => {
+  console.log("Message written.");
+}).catch(_ => {
+  console.log(`Write failed :-( try again: ${error}.`);
+});
+```
+
+### Security and permissions {: #security-and-permissions }
+
+The Chrome team has designed and implemented Web NFC using the core principles
+defined in [Controlling Access to Powerful Web Platform
+Features][powerful-apis], including user control, transparency, and ergonomics.
+
+As NFC expands the domain of information potentially available to malicious
+websites, the availability of NFC has been restricted to maximize users'
```

Comment:
I think it makes sense actually.
Because NFC expands the number of info available to websites, the Chrome team decided to restrict the availability of NFC in Web NFC.

Reply:
Tense shifts in a single sentence are an anti-pattern. In this case it prevented me from seeing what you were trying to say. The correction is simpler than I first thought.

* Change 'As' to 'Because' at the start.
* Change 'NFC has been restricted' to 'NFC is restricted'.

Reply:
Done.